README.md

```diff
@@ -273,6 +273,10 @@ plugin to your Babel config:
 - `server` (default: `true`) - When `true` adds `serverSideRequirePath` config.
 - `webpack` (default: `false`) - When `true` adds `webpackRequireWeakId` config.
 
+**Note:**
+
+If you're using babel + webpack with [syntax-dynamic-import](https://babeljs.io/docs/plugins/syntax-dynamic-import/) plugin and it causes your server-side-rendering dont render on the first request. Consider using [babel-plugin-dynamic-import-node](https://github.com/airbnb/babel-plugin-dynamic-import-node) plugin for your server build. It'll convert **import()** to **require()** and ensure synchronous rendering.  
```

Comment:
No I'm not saying using "babel-plugin-dynamic-import-node" (bpdin for short) alone will do the work. I am using babel-plugin-dynamic-import-node **with** the react-loadable/babel plugin **of course**.

But without babel-plugin-dynamic-import-node my first server-side-renders always return null for loadable-component as i explain above.